docs/zh/guide/doc-page.md

```diff
@@ -1 +1,49 @@
 # 正文页面
+
```

Comment:
May you should put these content to `use-mdx` part and rebase the master branch so you can get the latest docs.

Reply:
ok, i will modify this part

Reply:
done